hnn_core/parallel_backends.py

```diff
@@ -118,9 +120,19 @@ def simulate(self, net):
         dpls = []
 
         parallel, myfunc = self._parallel_func(self._clone_and_simulate)
-        sim_data = parallel(myfunc(net, idx) for idx in range(n_trials))
+        sim_data = parallel(myfunc(net, idx) for
+                            idx in range(n_trials))
 
         dpls = _gather_trial_data(sim_data, net, n_trials)
+
+        if postproc:
+            N_pyr_x = net.params['N_pyr_x']
+            N_pyr_y = net.params['N_pyr_y']
+            winsz = net.params['dipole_smooth_win'] / net.params['dt']
+            fctr = net.params['dipole_scalefctr']
+            for idx in range(n_trials):
+                dpls[idx].post_proc(N_pyr_x, N_pyr_y, winsz, fctr)
```

Comment:
do you think we could avoid the duplication of code by putting this inside `_gather_trial_data` with `postproc` as an argument?